apps/src/code-studio/signup.js

```diff
@@ -0,0 +1,145 @@
+import $ from 'jquery';
+
+window.SignupManager = function (options) {
```

Comment:
A pattern we've been following in code-studio is to [export everything to window in code-studio.js](https://github.com/code-dot-org/code-dot-org/blob/f19927a8af509e17ce4a2d980e6e353303c8c7b6/apps/src/code-studio/code-studio.js#L41-L56), so we can track globals.  This might also be a good fit for the `window.dashboard` namespace.